code/datums/mood_events/generic_negative_events.dm

```diff
@@ -113,6 +113,11 @@
 	description = "<span class='warning'>I'm missing my family heirloom...</span>\n"
 	mood_change = -4
 
+/datum/mood_event/healsbadman
+	description = "<span class='nicegreen'>I feel a lot healthier, but also like shit.</span>\n"
```

Comment:
wtf? doesn't it say it feels strange but good or something

Reply:
wtf why is everyone flipping around what it says. someone told me the complete opposite before

Reply:
i might be wrong

Reply:
`to_chat(user, "<span class='notice'>You start to smear [src] on yourself. It feels and smells disgusting, but you feel amazingly refreshed in mere moments.</span>")`

>you feel amazingly refreshed

Reply:
>feels disgusting >shit is disgusting >feels like shit

autism 1 - pkp 0

Reply:
1. bad smell/feel
2. effect kicks in milliseconds later
3. amazingly refreshed feel with no smell

Reply:
It feels terrible to smear that on you, but your body feels refreshed. Unless it erases your memories you're still going to remember it feeling awful to smear about even though you have no pains, aches or tiredness anymore.

Reply:
crazy idea: just change the desc to not cause a semantics fit

Reply:
smh spoiled my magic trick